src/redux/selectors/ws/get_execution_connection_state.js

```diff
@@ -0,0 +1,18 @@
+import _get from 'lodash/get'
+import { REDUCER_PATHS } from '../../config'
+
+const path = REDUCER_PATHS.WS
+
+const getExecutionConnectionState = (state) => {
+  const lostConnectionAt = _get(state, `${path}.execution.lostConnectionAt`, null)
+  const _connectionLostDurationMs = _get(state, `${path}.execution.connectionLostDurationMs`)
```

Comment:
use createSelector which touches only `execution` state, this will likely return new obj ref on any state change and not just execution